ARGO/src/GameScreen.cpp

```diff
@@ -145,6 +152,14 @@ void GameScreen::createEnemy()
 	m_entities.back().addComponent(new TagComponent(Tag::Enemy));
 	m_entities.back().addComponent(new ForceComponent());
 	m_entities.back().addComponent(new HealthComponent(Utilities::ENEMY_HP, Utilities::ENEMY_HP));
+	m_entities.back().addComponent(new VisualComponent("EnemyFast.png", m_renderer));
+}
+
+void GameScreen::createGoal()
+{
+	m_goal.addComponent(new TransformComponent(glm::vec2(Utilities::TILE_SIZE * 51, Utilities::TILE_SIZE * 36)));
```

Comment:
Again 51 and 36 are magic numbers. maybe make them a value in utilities because they are used in more than one place

Reply:
Already had it in Utilities, just forgot to update it XD